src/transformers/less.ts

```diff
@@ -20,10 +22,14 @@ const transformer: Transformer<Options.Less> = async ({
     ...options,
   });
 
+  const dependencies = imports.map((path: string) =>
+    isAbsolute(path) ? path : join(process.cwd(), path),
```

Comment:
Are the paths relative to the cwd? Something like `node_modules/...`?

Reply:
As described in #495 you currently a warning like this:
```
WARNING in ./app/App.svelte
Invalid dependencies have been reported by plugins or loaders for this module. All reported dependencies need to be absolute paths.
Invalid dependencies may lead to broken watching and caching.
As best effort we try to convert all invalid values to absolute paths and converting globs into context dependencies, but this is deprecated behavior.
Loaders: Pass absolute paths to this.addDependency (existing files), this.addMissingDependency (not existing files), and this.addContextDependency (directories).
Plugins: Pass absolute paths to fileDependencies (existing files), missingDependencies (not existing files), and contextDependencies (directories).
Globs: They are not supported. Pass absolute path to the directory as context dependencies.
The following invalid values have been reported:
 * "node_modules/package/file.less"
 @ ./app/App.svelte
 @ ./app/app.js
```
As you can see, the path is `node_modules/package/file.less` but should be `/home/user/project/node_modules/package/file.less`.

I based the fix on this code in `src/transformers/scss.ts`:
```ts
  const absoluteEntryPath = isAbsolute(result.stats.entry)
    ? result.stats.entry
    : join(process.cwd(), result.stats.entry);
```
Works for https://github.com/lundal/svelte-less-example atleast.